### ISSUE TITLE
Thoughts on recipe issue templates

### ISSUE THREAD

**Issue by milesfrain** [JordanMartinez/purescript-cookbook#185]
We currently have these two options:
* Bug report
* Fix Broken Recipe

What about an issue for improving a working recipe?

Should all of these be merged under something like "Recipe Issue"?

**Comment by JordanMartinez** [comment]
I think 'fix broken recipe' should be left as is. What you're proposing is a recipe improvement. The bug report should likely stay since a recipe (or this repo) could have a bug. Then again, I think we'd likely use the 'cookbook' label or 'improve-recipe' label.... Hm... I'm not sure anymore, haha.

**Comment by JordanMartinez** [comment]
Yeah, let's go with Recipe Issue. That can cover bugs, improvements, etc.
The `cookbook` label can refer to any "meta" things relating to this project or how we're doing things.
`recipe-request` will always refer to a recipe request.
